stream_alert/rule_processor/handler.py

```diff
@@ -25,21 +28,26 @@
 from stream_alert.shared.metrics import MetricLogger
 
 
+MAX_RECORD_SIZE = 1000 * 1000
+MAX_BATCH_SIZE = 500
+
+
 class StreamAlert(object):
-    """Wrapper class for handling all StreamAlert classificaiton and processing"""
+    """Fascade for handling StreamAlert classificaiton and processing"""
 
     def __init__(self, context, enable_alert_processor=True):
-        """
+        """Initializer
+
         Args:
-            context: An AWS context object which provides metadata on the currently
+            context (dict): An AWS context object which provides metadata on the currently
                 executing lambda function.
             enable_alert_processor (bool): If the user wants to send the alerts using their
                 own methods, 'enable_alert_processor' can be set to False to suppress
                 sending with the StreamAlert alert processor.
         """
         # Load the config. Validation occurs during load, which will
         # raise exceptions on any ConfigErrors
-        config = load_config()
+        self.config = load_config()
```

Comment:
If we're going to start saving this on init, can we move it to a class level property and only call `load_config` if the `StreamAlert._config` isn't populated?

Reply:
I'm actually not sure this needs to even be an instance attribute anymore, but I can still make that change